### PR TITLE
Content on pages

### PR DESCRIPTION
**Pull request by msecret**
Adds and changes content for the different sections.

@brittag you may want to review content part

Ref #309

### CODE REVIEW

File: static_src/components/app_page.jsx [hunk 1]
```diff
@@ -58,7 +58,7 @@ export default class AppPage extends React.Component {
         <div>
           <h2>{ this.state.app.name }</h2>
           <section className={this.styler('section-card')}>
-            <h3>About</h3>
+            <h3>About this</h3>
```
Comment (brittag) [inline static_src/components/app_page.jsx]: This should be "About this application"

File: static_src/components/app_page.jsx [hunk 2]
```diff
@@ -113,19 +113,22 @@ export default class AppPage extends React.Component {
           </section>
           <section className={this.styler("section-card")}>
             <h3>Services</h3>
-            <p>To bind or unbind a service instance to an app view<a
+            <p>To bind or unbind a service instance to an app, follow the<a
               href="https://docs.cloud.gov/apps/managed-services/#bind-the-service-instance"
               target="_blank">
-              <span>&nbsp;</span>managed services guide</a> for more information.
+              <span>&nbsp;</span>managed services guide</a>.
```
Comment (brittag) [inline static_src/components/app_page.jsx]: Hm, this is rendering without a space:

screen shot 2016-07-08 at 11 34 57 am

File: static_src/components/route_list.jsx [hunk 3]
```diff
@@ -77,10 +77,10 @@ export default class RouteList extends React.Component {
       <div className={ this.styler('tableWrapper') }>
         { content }
         <aside>
-          <p>To modify, create or delete a route, view <a
+          <p>To create, modify, or delete a route for an app, follow the <a
             href="https://docs.cloud.gov/apps/custom-domains/#application-routes"
             target="_blank">
-            <span>&nbsp;</span>application routes</a> for more information.
+            <span>&nbsp;</span>application routes</a>.
```
Comment (brittag) [inline static_src/components/route_list.jsx]: This should be "application routes guide"

**Comment by brittag** [comment]
Yay, thanks for putting in this copy! I added a couple little notes.

The styling of the temporary "how to add/remove roles" section is a little confusing since the bulleted list has different formatting:

screen shot 2016-07-08 at 11 37 39 am

It'd be good for that to be consistent. Ideally I'd even put the "how to add/remove roles" section into a gently styled alert box, to help communicate that these are temporary instructions making up for a temporarily confusing interface. :)

**Comment by brittag** [comment]
Thanks, these content updates look good to me now!

**Comment by jeremiak** [comment]
Looks good to me, merging!

**Pull request merged**